common/utils/api-utils.js

```diff
@@ -50,10 +53,15 @@ export const get = async (path, { token, parameters } = {}, axiosClient = Operat
       headers: setAuthorizationHeader(token),
       cancelToken: abort.token,
       params: parameters,
+      paramsSerializer: parameters_ => qs.stringify(parameters_, { arrayFormat: 'repeat' }),
     })
     .then(response => {
       clearTimeout(connectionTimeout);
       return response;
+    })
+    .catch(error => {
+      clearTimeout(connectionTimeout);
+      return error.response.data;
```

Comment:
Can you just return the error? As it stands, we've so far been good about handling every exception on a case-by-case basis.